src/mudlet.cpp

```diff
@@ -672,242 +376,613 @@ mudlet::mudlet()
+            mTranslationsMap.insert(languageCode, translation);
+        } else {
+            // This is very unlikely to be reached as it means that a file that
+            // matched the naming to be a Mudlet translation file was not infact
+            // one...
+            qDebug().noquote().nospace() << "mudlet::scanForMudletTranslations(...) INFO - no Mudlet translation found for locale code:\"" << languageCode << "\".";
+        }
+        delete pMudletTranslator;
+    }
+}
+
+// To be used AFTER scanForMudletTranslations(...) has been called, this will
+// insert the corresponding Qt system translation pathFileNames (path to
+// filenames: qt_xx.qm or qt_xx_YY.qm) into the entries in the mTranslationsMap
+// QHash.
+void mudlet::scanForQtTranslations(const QString& path)
+{
+    mQtTranslationsPathName = path;
+    qDebug().nospace().noquote() << "mudlet::scanForQtTranslations(\"" << path << "\") INFO - Seeking Qt translations files in given location.";
+    QMutableHashIterator<QString, translation> itTranslation(mTranslationsMap);
+    while (itTranslation.hasNext()) {
+        itTranslation.next();
+        const QString languageCode = itTranslation.key();
+        auto pQtTranslator = new QTranslator();
+        QString translationPathFileName(QStringLiteral("%1/qt_%2.qm").arg(path, languageCode));
+        if (pQtTranslator->load(translationPathFileName, path)) {
+            qDebug().noquote().nospace() << "mudlet::scanForQtTranslations(...) INFO - found a suitable Qt translation for locale code:\"" << languageCode << "\".";
+            /*
+             * Unfortunately, success in this operation does not mean that
+             * the qt_xx_YY.qm translation file has been located and used as
+             * (bool) QTranslator::load(const QString &filename,
+             *                          const QString &directory = QString(),
+             *                          const QString &search_delimiters = QString(),
+             *                          const QString &suffix = QString())
+             * call can forget about both the _YY and even the _xx if filenames
+             * with those eleements are not found but a less detailed filename
+             * IS detected.
+             * As I noted for https://bugreports.qt.io/browse/QTBUG-64416 there
+             * is also the possibilty to use QTranslator::load(...) form that
+             * looks in the user settings for their locale preferences - which
+             * can include several locale specifiers and then the Qt code will
+             * look the first one it can find - but it does not report back
+             * which one that was.
+             *
+             * So although we can note the success pathFileName it might not be
+             * what it appears to be - it could be something less specific, e.g.
+             * /path/to/Qt lib translations/qt_en.qm
+             * even if we asked for
+             * /path/to/Qt lib translations/qt_en_GB.qm...
```

Comment:
Can we cut down on this 20 line comment?

Reply:
It ***is*** documenting a gotcha which someone perusing the code might not be aware off - I'll see if I can précis it a bit...

Reply:
I hadn't completely forgotten this - I will get to it...